src/main/java/com/learning/pages/google/GoogleSearchPage.java

```diff
@@ -0,0 +1,62 @@
+package com.learning.pages.google;
+
+import com.learning.configuration.PropertiesReader;
+import com.learning.pages.PageBase;
+import java.util.Objects;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class GoogleSearchPage extends PageBase {
+
+  @FindBy(name = "q")
+  private WebElement searchBox;
+  @FindBy(name = "btnK")
+  private WebElement searchButton;
+
+  private final PropertiesReader propertiesReader;
+  private static final String GOOGLE_URL_PROPERTIES_NAME = "googleURL";
+
+  public GoogleSearchPage(WebDriver driver, PropertiesReader propertiesReader) {
+    super(driver);
+    this.propertiesReader = propertiesReader;
+  }
+
+  public void openGoogleWebsite(){
+    driver.get(getGoogleURL());
+    waitForPageToLoad();
+  }
+
+  public void searchGoogle(String searchPhrase) {
+    searchBox.sendKeys(searchPhrase);
+    searchButton.submit();
+  }
+
+  public WebElement getGoogleSearchTextBox() {
```

Comment:
don't return WebElement from PageObject, use "tell, don't ask" approach, move whatever logic you have into dedicated method instead

Reply:
Sure. Also I don't use it in test after changes. To delete.